commons/ava_testsuite/rpc_manager.go

```diff
@@ -0,0 +1,147 @@
+package ava_testsuite
```

Comment:
Thoughts about this file as a whole: AFAICT it looks like:
1. These methods are meant to be run in conjunction, one after each other
2. They're meant to be run exactly once
3. They're only used in one place

Since this is true, I don't think we want to have this logic as a separate struct - instead, it seems like we'd want to have a `initializePChainAccount` function in the 5-node staking network tests, and that function should do the equivalent of both of these.

Reply:
refactored to high level gecko client